src/pages/palanbyulChoi/Main/Love.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+class Love extends Component {
+  changeFeedLove = () => {
+    this.props.changeFeedLove(this.props.feedId);
+  };
+  render() {
+    return (
+      <>
+        <i
+          className={this.props.isLiked ? 'fa fa-heart' : 'far fa-heart'}
+          onClick={this.changeFeedLove}
+        />
+      </>
+    );
+  }
+}
+export default Love;
```

Comment:
- 코드에 줄 띄우기만 추가해주셔도 가독성이 많이 좋아질 수 있습니다.
```suggestion
import React, { Component } from 'react';

class Love extends Component {
  changeFeedLove = () => {
    this.props.changeFeedLove(this.props.feedId);
  };
  
  render() {
    return (
      <>
        <i
          className={this.props.isLiked ? 'fa fa-heart' : 'far fa-heart'}
          onClick={this.changeFeedLove}
        />
      </>
    );
  }
}

export default Love;
```

Reply:
넵 다른 부분도 수정하도록 하겠습니다